keyboards/splitkb/aurora/corne/readme.md

```diff
@@ -0,0 +1,48 @@
+![Aurora Corne](https://raw.githubusercontent.com/splitkb/qmk_firmware/assets/aurora/corne/corne.jpg)
```

Comment:
Could you rehost this on Imgur?

Reply:
Any reason to _specifically_ use Imgur?

The docs say to use an external service to host images - I understood this to mean that they can not be in the main repo, which is understandable as it would greatly increase the repo's size.

The way I did it here (and for the Aurora Sweep) is by placing them in a fork of the repo so they do not add any size to the main repo, while still providing us the convenience of having them close to the code itself. I though this was acceptable as there are already 370+ images linked in this way in the repo. Is there anything wrong with this?

Reply:
Imgur allows you to resize the image simply by adding a character to the end of the filename in the URL, this way if someone just uploads their full res photo and puts it in the readme, it can be easily shrunk down to a more reasonable (file)size.
https://api.imgur.com/models/image#thumbs

Reply:
Yeah, I noticed that - it's why I already scaled down all images to a width of 1000ish px. :smile:

Reply:
also, `githubusercontent.com` isn't exactly permanent. If you change the repo, delete it, or github just decides to change how something works, it will cause the url to stop working properly. And unfortunately, none of these listed scenarios are theoretical.

Reply:
@splitkb this is why this has not been merged yet. We specifically ask for imgur images because they are permanent, as drashna says, githubusercontent.com is transient and images can disappear. It's a standard we ask for clearly in the PR checklist.

Reply:
All images have been rehosted on Imgur